feature/platform/tests/power_admin_down_up_test/power_admin_down_up_test.go

```diff
@@ -24,6 +24,22 @@ func TestFabricPowerAdmin(t *testing.T) {
 	dut := ondatra.DUT(t, "dut")
 	fs := components.FindComponentsByType(t, dut, oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_FABRIC)
 
+	removable_fabrics := make([]string, 0)
+	for _, f := range fs {
+		compMtyVal, compMtyPresent := gnmi.Lookup(t, dut, gnmi.OC().Component(f).Empty().State()).Val()
+		if compMtyPresent && compMtyVal {
+			continue
+		}
+		if gnmi.Get(t, dut, gnmi.OC().Component(f).Removable().State()) {
```

Comment:
Looks like this check already exists just a few lines below.

https://github.com/openconfig/featureprofiles/blob/65ec620e7aa48b80d10aa46ea3f0043e83978c75/feature/platform/tests/power_admin_down_up_test/power_admin_down_up_test.go#L30-L32

Is there a reason we need another one?

Reply:
Hi,
Yes there is need to add this check. It is considering all the sibs even which are not really installed in the device for further test. So added this empty check which will consider only the sibs which were there in the device.

Thanks

Reply:
Ok this is added as part of pull - https://github.com/openconfig/featureprofiles/commit/d93f6e796eb80fc4d2fff96d545f27208eb5aa1d
which is commited on jul21

But before that these lines were not present I raised pull request on june 28. Should I need to abort it then??

Reply:
Corrected the code accordingly. Empty check should be done initially before removable state check. Please check.